c++/src/capnp/serialize.h

```diff
@@ -119,6 +119,10 @@ size_t expectedSizeInWordsFromPrefix(kj::ArrayPtr<const word> messagePrefix);
 // using the full KJ async infrastructure would be too difficult. Each time bytes are received,
 // use this function to determine if an entire message is ready to be parsed.
 
+kj::Array<kj::byte> segmentsTable(kj::ArrayPtr<const kj::ArrayPtr<const word>> segments);
```

Comment:
I think this should return `Array<word>`. Caller can always use `.asBytes()` on it if desired, but returning `Array<word>` promises that it'll be properly aligned, which in helpful.

Reply:
Done. I think I have the math right?